### PR TITLE
Tweak wording about README.dist.md

### PR DESCRIPTION
**Pull request by pyrech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,10 +51,12 @@ Alternatively, you can configure
 `infrastructure/docker/services/router/generate-ssl.sh` to create your own
 certificate. Then you will have to add it to your computer CA store.
 
-## Final documentation
+## Usage documentation
 
-This starter-kit embeds a [README.dist.md](./README.dist.md) you can use as the
-base of the final README.md of your project:
+We provide a [README.dist.md](./README.dist.md) to explain what everybody wiil need
```
Comment (lyrixx) [inline README.md]: I wiiiiiiiiiiiiiiiiil survive :)

Comment (lyrixx) [reply README.md]: You can remove `will` anyway :)

Comment (pyrech) [reply README.md]: Done

**Comment by lyrixx** [comment]
Thanks @pyrech

**Pull request merged**